app/assets/icons/close.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const CloseIcon = ({ size = 24, color = "#333", className}) => (
+  <svg className={className} width={size} height={size} viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+    <g fill="none" fill-rule="nonzero" stroke={color} stroke-linecap="square" stroke-width="2">
```

Comment:
Need to convert from svg to react attribs:
`fillRule` instead of `fill-rule`, `strokeWidth` instead of `stroke-width` etc.

Reply:
can we move to svg sprite instead? if anyone wants to pair on this, I am available. If nothing major comes up in vikatan.

Reply:
This has to be moved once the svg sprite PR goes in

Reply:
Moved to svg sprites in Ahead already, can do in Malibu as well